storage/gc_queue_test.go

```diff
@@ -167,14 +169,14 @@ func TestGCQueueProcess(t *testing.T) {
 		{key5, ts2, true, false},
 		// For key6, expect no values to GC because most recent value is intent.
 		{key6, ts1, false, false},
-		{key6, ts5, true, true},
+		{key6, ts5, false, true},
```

Comment:
It was a mistake previously, because it didn't match the comment, which specifies a value, not a deletion.